CITATION.cff

```diff
@@ -0,0 +1,16 @@
+cff-version: 1.2.0
+title: "conda: A system-level, binary package and environment manager running on all major
+  operating systems and platforms."
+message: >-
+  If you use this software, please cite it using the metadata from this file.
+type: software
+authors:
+  - name: "The conda development team"
```

Comment:
Let's "contributors" here to acknowledge all kinds of contributions, not just code.
```suggestion
  - name: "conda contributors"
```